.github/workflows/test_pr.yml

```diff
@@ -34,4 +34,25 @@ jobs:
         with:
           args: deploy -P roster-pr --only hosting:project-dashboard-pull-request
         env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
\ No newline at end of file
+        FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+
+
+# Only run the coverage once
+      name: get value percentage and print
+      run: |
+        PERCENTAGE=$(npm test -- --coverage --coverageReporter='text-summary' | grep 'All files' | cut -d '|' -f 5)
+        echo $PERCENTAGE
+        echo "PERCENTAGE=$(echo ${TOKENS[2]})" >> $GITHUB_ENV
```

Comment:
TOKENS is not defined and not needed. cut on previous command takes the substring

Reply:
You are adding PERCENTAGE to env but later you are using COVERAGE